Index.html

```diff
@@ -217,27 +220,29 @@ <h1 class="bottom-line">Skills</h1>
         <hr>
     </ul>
     </article>
-    <article class="header-img">
+
+    <footer class="header-img">
         <h1 class="index-h1">Contact me</h1>
-        <p>If you have an application you are interested in developing,
+        <p class="text">
+            If you have an application you are interested in developing,
            a feature that you need built or a project that needs coding.
            I'd love to help with it 
         </p>
-        <form class="form-control">
+        <form action="https://formspree.io/f/{form_id}" method="POST">
```

Comment:
- I don't think your formspree has been set up properly. Can you set it up. When you do, it will generate an `id` that you can use in place of `/{form_id}` that you have used.